aioshelly/coap.py

```diff
@@ -79,12 +170,15 @@ def __init__(self, message_received=None):
         self._message_received = message_received
         self.subscriptions = {}
         self.transport: Optional[asyncio.DatagramTransport] = None
+        self.querier = None
 
     async def initialize(self):
```

Comment:
We should add a flag here (default can be true) so that we can later disable/enable querier without releasing a new version if we have problems with it.

Reply:
Moved start/stop to HA side

Reply:
Done